variables.tf

```diff
@@ -271,5 +281,5 @@ variable "webhooks" {
     })
   }))
   default     = []
-  description = "List of webhook configurations"
+  description = "List of webhook configurations."
```

Comment:
I am more in favor of avoiding the dot (here, signifying the end of a sentence) in such use cases, for the same reason as leaving it out in the commit message. When there are more sentences, I use it. But when there's only one, this extra character serves no purpose, IMO.

Reply:
I think it is OK to leave it here